### DISCUSSION TITLE
Lost Your Progress?

### DISCUSSION THREAD

**Discussion by PavelZinchenko** [PavelZinchenko/event-horizon-frontier-builds#69] [General]
Sometimes that occurs, and it can be quite distressing. There are instances where restoration might not be feasible. However, if I'm unable to restore your original save, I can create a new one that closely resembles what you had. Just drop a post here, providing as much detail as possible about what you lost, and we'll work together to recreate it.

**Comment by AceCripth** [comment]
Hello i just created an account for this thing, since i just lost my 5 years of progress by just updating the app from github that was before the rework of the gold+ modules. It happens when i update that newest patch where its rear admirals were changed and addition to new function of scientist npc. My progress on that save was all of the ships are maxed and some are expanded (mostly cap ships) except for invader since i dont have it, i also have lots of 20+ gold modules from weapons to defense and s1 drones that literally took me months to fully 20+ gold that 20 modules of drones, all of the starbases with new starbase of course with the new sattelite, had like lots of stars from grinding my way up to 900+ levels, and lastly the level that im currently before i lost that progress is at around 910. When i lost that progress by just updating the game i got mad and confused, then move on but when i start to get that progress back i felt burnout to get to that progress again since i've been playing that game for 5 years just to have my progress gone in a matter of seconds i hope this details help me to recreate my progress back, thanks (sorry if my english is bad).

**Reply by PavelZinchenko** [reply]
Hello. Could you share your email with me? I need it to send you a savegame file

**Reply by AceCripth** [reply]
Yes its Acecripth14@gmail.com

**Reply by PavelZinchenko** [reply]
Done

**Reply by Kryptonit2010** [reply]
I lost progress and I have a video of how I played the evidence is very offensive I'm hail and already lost 3 accounts I'm devastated I had level 90-94 definitely don't remember 360 stars Jan salo and his ship and all ships of level 80 (I don't remember the name) artificial intelligence module 3 pieces and heavy lasers also 12 neutron lasers

**Reply by Kryptonit2010** [reply]
Sorry for my bad English

**Comment by NH23727** [comment]
Hello, I sent an email detailing my save to help.eventhorizon@gmail.com before seeing this discussion post, would be glad if you can restore my save. I will post the details again on github if requested.

**Reply by PavelZinchenko** [reply]
Hello. I've been getting quite a lot such emails recently. Which one is from you?
Looks like I found it.

**Reply by memetallica69** [reply]
@PavelZinchenko I lost my save data on my iPhone is there anything you or me could do? My email is coxevan579@gmail.com

**Comment by skirmsher** [comment]
Yo i recently made a oopsie, wanted to try the beta fetures and after i was done i tried to go back to the normal verison by reinstalling the game and my game files got deleted!!! and now im really anxious so try to be urgent with this 😭😭😭😭😭 i can send my email via discord aka AlmostThrashed

**Reply by PavelZinchenko** [reply]
Hey. Just write to help.eventhorizon@gmail.com and describe your progress. But just so you know, I can only create a new save for the beta version.

**Reply by skirmsher** [reply]
I sent a email

**Comment by skirmsher** [comment]
Yo i sent a email quite a time ago and still no response...

**Comment by Tree3324417** [comment]
There is someone i know (email: trevmarz13@gmail.com) who sent you an email regarding save file format upgrade making him lose save in Event Horizon (not frontier, but still, he wanted save restoration so i figured this is the best place to ask for attention). Can you check it out?

**Comment by skirmsher** [comment]
Sent a email to help.eventhorizon@gmail.com months ago and still havent gotten a recovered file or something close reseambling it, pavel please make a auto genereated file for the level i was on, cause now months later i dont remember much or read through like 1000 letters to get exactly as i remembered it

**Comment by Khatt5** [comment]
good morning, afternoon or evening  

I recently saw a post that said they could restore a saved game from the game Event Horizon Frontier and I would like to ask for your help to recover my saved game. 


This game had all the ships unlocked and at the maximum level you also got a great variety of all the modules  

I had also completed all 10 stages of each of the arena duels. 

I was also at level 666 or something like that, I had a great job with 97,639 stars and about 1 million credits. 

I forgot to mention that I had upgraded all the modules to the maximum with the scientist and I had at least 23 blueprints for each module, I had at least 35 for each satellite and quite a few rare tokens and resources. 

It also had the ships from the Christmas event and all the modules and plans for the event. 

This is what I remember, it was from version 3.0.0 but I would really like it if they restore my save it was for version 3.1.0 

without more to say thank you 

post: lasñ version 3.1.0 I got it from its official site

**Reply by Cruxean** [reply]
uhh, i lost my cloud save file account, its on level 180-190, and it has covid 6019 and other capital ships, can you just send me a working regular save file with base covid 6019 and 1000 stars as compensation?

my email is: pm6632601@gmail.com
